src/cpu/gemm_convolution_utils.cpp

```diff
@@ -452,7 +452,7 @@ void im2col_dt_3d(const conv_gemm_conf_t &jcp,
     bool with_input_zp = input_zp != nullptr;
 
     if (sd == 1 && sh == 1 && sw == 1 && dd == 1 && dh == 1 && dw == 1)
-        parallel_nd(jcp.kd, jcp.kh, jcp.kw, jcp.ic,
+        parallel_nd_for_im2col(jcp.kd, jcp.kh, jcp.kw, jcp.ic,
```

Comment:
minor: I guess `for` is redundant in this name and in other routines.

Reply:
Fixed. I moved these methods to dnnl_thread.hpp and added "_templ" suffix